Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -387,8 +393,11 @@ namespace AZ
                                 if (TestOcclusionCulling(worklistData, visibleEntry) == MaskedOcclusionCulling::CullingResult::VISIBLE)
 #endif
                                 {
-                                    numDrawPackets += AddLodDataToView(c->m_cullData.m_boundingSphere.GetCenter(), c->m_lodData, *worklistData->m_view);
-                                    ++numVisibleCullables;
+                                    #ifdef AZ_CULL_DEBUG_ENABLED
+                                        ++numVisibleCullables;
+                                        numDrawPackets += // Return value of AddLodDataToView
+                                    #endif
+                                    AddLodDataToView(c->m_cullData.m_boundingSphere.GetCenter(), c->m_lodData, *worklistData->m_view);
```

Comment:
We could also just add an else block:
```
#if AZ_CULL_DEBUG_ENABLED
    numDrawPackets += AddLodDataToView(c->m_cullData.m_boundingSphere.GetCenter(), c->m_lodData, 
        *worklistData->m_view);
    ++numVisibleCullables;
#else
    AddLodDataToView(c->m_cullData.m_boundingSphere.GetCenter(), c->m_lodData, *worklistData->m_view);
#endif
```

Reply:
> We could also just add an else block:
> 
> ```
> #if AZ_CULL_DEBUG_ENABLED
>     numDrawPackets += AddLodDataToView(c->m_cullData.m_boundingSphere.GetCenter(), c->m_lodData, 
>         *worklistData->m_view);
>     ++numVisibleCullables;
> #else
>     AddLodDataToView(c->m_cullData.m_boundingSphere.GetCenter(), c->m_lodData, *worklistData->m_view);
> #endif
> ```

The else block is brittle because it requires 2 copies of the function call to be updated any time it is changed.

Reply:
Yeah, I didn't do #else because the annoying bug possibility was too high.
I change to rgba16f proposal on this with a comment that maybe_unused does not need to be fixed.